lib/mockBuilder.js

```diff
@@ -13,7 +13,7 @@ function mockBuilder (schemaCode, customMock) {
   const mockSchema = {}
   for (const type of Object.keys(schema)) {
     const mockField = memoizedField(type, customMock, schema)
-    if (JSON.stringify(mockField) !== JSON.stringify({})) {
+    if (Object.keys(mockField).length > 0) {
```

Comment:
I changed this check because a cycle in type references will produce a circular structure, which cannot be serialized to JSON. `JSON.stringify` function emits enumerable properties, and `Object.keys` lists enumerable property names; so the new version of the check should have the same result.